hugegraph-common/src/test/java/org/apache/hugegraph/unit/util/StringUtilTest.java

```diff
@@ -17,16 +17,16 @@
  * under the License.
  */
 
-package com.baidu.hugegraph.unit.util;
+package org.apache.hugegraph.unit.util;
 
 import java.util.ArrayList;
 import java.util.List;
 
+import org.apache.hugegraph.util.StringUtil;
+import org.apache.hugegraph.util.StringUtil.Chars;
 import org.junit.Test;
 
-import com.baidu.hugegraph.testutil.Assert;
-import com.baidu.hugegraph.util.StringUtil;
-import com.baidu.hugegraph.util.StringUtil.Chars;
+import org.apache.hugegraph.testutil.Assert;
```

Comment:
should `25~26 + 29` be together?  

or we need use check-style plugin to auto format it